model/index.js

```diff
@@ -27,23 +27,23 @@ let model = {
   },
   put: (query) => {
     return new Promise((resolve, reject) => {
-      db.query(query, (err) => {
+      db.query(query, (err, data) => {
         if (err) {
           reject(err);
         } else {
-          resolve();
+          resolve(data.rows[0]);
         }
       });
     });
   },
   delete: (args) => {
-    let query = 'DELETE FROM seller_info WHERE item_id = $1';
+    let query = 'DELETE FROM seller_info WHERE item_id = $1 RETURNING item_id';
     return new Promise((resolve, reject) => {
-      db.query(query, args, (err) => {
+      db.query(query, args, (err, data) => {
```

Comment:
instead of an error first callback within a Promise, couldn't you just do db.query().then(success).catch(err)?

Reply:
You are 100% correct, I will refactor.